gdk4/src/display.rs

```diff
@@ -58,6 +61,16 @@ impl<O: IsA<Display>> DisplayExtManual for O {
         }
     }
 
+    fn get_setting(&self, name: &str, value: &glib::Value) -> bool {
```

Comment:
This should probably be

```suggestion
    fn get_setting(&self, name: &str) -> Option<glib::Value> {
```

Otherwise the value needs to be `&mut` as the function writes the result into it.